programmers/난이도별/level03.베스트앨범/sangmandu.py

```diff
@@ -0,0 +1,23 @@
+'''
+https://programmers.co.kr/learn/courses/30/lessons/42579
+베스트앨범
+딕셔너리로 해시 기능 이용. 수가 많은 장르와 각 장르별 순위를 매기기 위해 두 가지 딕셔너리 사용
+'''
+
+def solution(genres, plays):
+    rank, chart = {}, {}
+    for idx, val in enumerate(zip(genres, plays)):
+        a, b = val[0], val[1]
+        rank.setdefault(a, 0)
+        rank[a] += b
+        chart.setdefault(a, [])
+        chart[a].append((idx, b))
+    best = sorted(rank.items(), key=lambda x : x[1], reverse=True)
+    answer = []
+    for i in best:
+        answer.extend(sorted(chart[i[0]], key=lambda x : (-x[1], x[0]))[:2])
+    return [a[0] for a in answer]
```

Comment:
```suggestion
    return [a for [a] in answer]
```
아마도 이렇게 표현할 수 있을 것 같아요!

Reply:
오?? 저러면 [a]에 대한 a가 0번째 값을 의미하게 되나요??

Reply:
@sangmandu 맞습니다~

Reply:
저렇게 표현하는 것의 장점이 있나요?? 그리고 1번째 값도 표현할 수 있나요??

Reply:
@sangmandu 
```python
    return [a for [, a] in answer]
```
1번째 값은 이렇게 표현할 수 있어요!
음.. 특별한 장점은 인덱스에 대한 변수 명을 원하는대로 지정할 수 있다는 점이 아닐까 싶어요 ㅎㅎ

Reply:
감사합니다